com.jetbrains.PyCharm-Community.yaml

```diff
@@ -17,95 +17,104 @@ finish-args:
   - --filesystem=xdg-run/keyring
   - --device=all
   - --env=PYCHARM_JDK=${FLATPAK_DEST}/pycharm/jre64/
+  - --env=PATH=/app/bin:/usr/bin:/app/extensions/bin:app/packages/bin
```

Comment:
There’s a typo in the last element: `app/packages/bin` → `/app/packages/bin`

Reply:
Thanks. Fixed now.